src/editor-finder.ts

```diff
@@ -72,16 +72,18 @@ export class EditorFinder implements IDisposable, IDebuggerEditorFinder {
    *
    * @param debugSessionPath The path for the current debug session.
    * @param source The source to find.
+   * @param focus - Set to true to focus on the relevant cell. Default to false.
    */
   find(
     debugSessionPath: string,
-    source: string
+    source: string,
+    focus: boolean
```

Comment:
This looks like we could move the focus to a separate plugin, but it would be fine to keep it like this for now and refactor after.